### ISSUE TITLE
Dropdown accessibility issue

### ISSUE THREAD

**Issue by smartm0use** [Semantic-Org/Semantic-UI#5272]
Here is the code that we get when the Semantic-UI dropdown is rendered:

http://jsfiddle.net/cshfb65c/1/

The problems is that the `input` tag should have `title` attribute, according to the [WCAG 2.0 specifications](https://achecker.ca/checker/).

How can I fix this?

**Comment by Banandrew** [comment]
Hi @smartm0use,

You can initialize a dropdown on [pre-existing HTML](https://semantic-ui.com/modules/dropdown.html#/usage) instead of initializing it on `select` element directly. This way you can add a `title` to the `input`.

Thank you for posting, but although it’s a valid usage question, we’ve limited GitHub Issues to bug reports and feature requests, keeping the board more manageable for maintainers; see the [contributing guidelines](https://github.com/Semantic-Org/Semantic-UI/blob/master/CONTRIBUTING.md) for more information on what kind of posts should find themselves into the GitHub Issues board.

To get answers or feedback that might allow you to repost this issue, please use one of our other support resources:
* [Gitter chatroom](https://gitter.im/Semantic-Org/Semantic-UI)
* [Official forums](http://forums.semantic-ui.com)
* [StackOverflow](http://stackoverflow.com/questions/tagged/semantic-ui)

**Comment by smartm0use** [comment]
I thing it could be an issue. Anyway the element

`<input class="search" autocomplete="off" tabindex="0">`

is added only when the dropdown/select is used in a modal.
Do you know the reason or can you help me to figure out why this happens?

**Comment by Banandrew** [comment]
@smartm0use Show it to me in a JSFiddle please.

**Comment by smartm0use** [comment]
I solved forking Semantic-UI and adding the following change:

![screenshot from 2017-04-13 16 02 49](https://cloud.githubusercontent.com/assets/5399265/25008082/e35e7b54-2062-11e7-94fd-a1e914f615e3.png)

**Comment by Banandrew** [comment]
@smartm0use Now I get it. The accessibility of the framework has been discussed before here #348, the _tl;dr_ version of it is that it’s out of the framework’s scope, and is a developer’s responsibility if they want their project to be compliant with the standard, so I don’t think the PR will accomplish anything—the issue is much bigger than that, and I’d recommend to give that thread a read to understand the problem with SUI’s accessibility better.
